ios/ReactNativeAudioToolkit/ReactNativeAudioToolkit/AudioPlayer.m

```diff
@@ -162,6 +162,12 @@ - (NSURL *)findUrlForPath:(NSString *)path {
     ReactPlayer *player = [[ReactPlayer alloc]
                         initWithPlayerItem:item];
 
+    // If we don't set this property the player appears to play
+    //(so we update currentTime etc) but the system has actually paused playback to buffer
+    if ([player respondsToSelector:@selector(setAutomaticallyWaitsToMinimizeStalling:)]) {
```

Comment:
Using `respondsToSelector` is the cleaner way of gating version specific code. Its a way of checking an object _can do_ something rather than checking a more arbitrary version number. Previously (in 68166e0d6069ff520b086b584a21b8a1c66c87c2) we did `[[[UIDevice currentDevice] systemVersion] floatValue] > 10`, which works but isn't quite as neat.